modules/magniteAnalyticsAdapter.js

```diff
@@ -633,6 +658,47 @@ const handleBidWon = args => {
   addEventToQueue({ bidsWon: [bidWon] }, bidWon.renderAuctionId, 'bidWon');
 }
 
+window.addEventListener('dmWebVitals', eventObj => {
+  let dmWebVitals = eventObj.detail;
+  dmWebVitals.accountId = accountId;
+  addEventToQueue({ dmWebVitals }, dmWebVitals.auctionId, 'webVital');
+});
+
+const countPendingEvents = (missedEvents, pendingEvents) => {
+  Object.keys(pendingEvents).forEach(event => {
+    if (Array.isArray(pendingEvents[event] && pendingEvents[event].length)) {
```

Comment:
I think there's an error here. there should be a closing `)` after `Array.isArray(pendingEvents[event]` ... I am testing this by refreshing the page an instant after the gam render and I saw that the event didn't get attached to the `missedEvents` object.

Reply:
ooooooooo wtf i am an idiot!

Reply:
fixed